package.json

```diff
@@ -4,7 +4,7 @@
   "description": "AWS, Azure, GCP, Oracle, GitHub security scanning scripts",
   "main": "index.js",
   "scripts": {
-    "test": "mocha './**/*.spec.js'",
+    "test": "nyc --reporter=text mocha './**/*.spec.js'",
```

Comment:
This is similar to the `test-cov` command below that already exists. Could we keep `test` as just for the mocha runs and leave `test-cov` for the coverage report?

Reply:
I have removed the nyc related changes